src/i18n/locales/en/translation.json

```diff
@@ -205,7 +205,8 @@
       },
       "language": {
         "title": "Language",
-        "subtitle": "Alby goes international! help us translate Alby in your language"
+        "subtitle": "Translations are not 100% finished yet.",
```

Comment:
```js
{
  "gdpr_policy_links": "I agree to <link1>Privacy Policy</link1> and <link2>Terms and Conditions</link2>",
  "gdpr_required_msg": "Please accept Privacy Policy"
}
```

This looks like we wouldn't need `link` and `subtitle` but only `subtitle` if we use this right. 
I hope we can do this:
```js
{
  "subtitle": "Translations are not 100% finished yet. <link>Help us translate Alby into your language!</link>",
}
```